js/options.js

```diff
@@ -49,7 +49,68 @@ document.addEventListener('DOMContentLoaded', function () {
     });
   }
 
+  // Helper for export function
+  let textFile = null;
```

Comment:
Using `let` is great, but it reduces the browser versions we support.

Chrome: 49+ (March 2, 2016)
Firefox: 44+ (Jan 26, 2016)

Is that something we want to do?

Reply:
Maybe, We don't support before Firefox 53. 
But we do support all versions of Chrome. So I will remove it.